ql/src/test/queries/clientnegative/alter_external_acid.q

```diff
@@ -4,6 +4,6 @@ set hive.txn.manager=org.apache.hadoop.hive.ql.lockmgr.DbTxnManager;
 
 create external table acid_external (a int, b varchar(128)) clustered by (b) into 2 buckets stored as orc;
 
-alter table acid_external set TBLPROPERTIES ('transactional'='true');
+alter table acid_external convert to Acid;
```

Comment:
do we have usecase that covers conversion to "insert-only"?

Reply:
Modified a insert-only q test, to have this covered, added explain for it as well